bot/build.sh

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env bash
+#
+# script to build the EESSI software layer. Intended use is that it is called
+# by a (batch) job running on a compute node.
+#
+# This script is part of the EESSI software layer, see
+# https://github.com/EESSI/software-layer.git
+#
+# author: Thomas Roeblitz (@trz42)
+#
+# license: GPLv2
+#
+
+# ASSUMPTIONs:
+#  - working directory has been prepared by the bot with a checkout of a
+#    pull request (OR by some other means)
+#  - the working directory contains a directory 'cfg' where the main config
+#    file 'job.cfg' has been deposited
+#  - the directory may contain any additional files references in job.cfg
+
```

Comment:
let's add a `set -e` here to stop the script as soon as something failed?

Reply:
Addressed in https://github.com/EESSI/software-layer/pull/233/commits/d08bd211ed6c23c5b917b01bd01ef75a5cb7b1b9